assets/src/edit-story/components/colorPicker/useKeyMoveStop.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { useKeyDownEffect } from '../keyboard';
+
+function useKeyMoveStop(ref, onMove) {
+  useKeyDownEffect(ref, 'left', () => onMove(-0.01), [onMove]);
+  useKeyDownEffect(ref, 'right', () => onMove(0.01), [onMove]);
```

Comment:
Fixing bug in 710b51c0 as [mentioned by @spacedmonkey](https://github.com/google/web-stories-wp/pull/497#issuecomment-596438886) (bug obviously introduced in [this refactor](https://github.com/google/web-stories-wp/pull/497/commits/3391b65d58ad518c7003c8a315513305a9af81d4#diff-1c08caaa370a6aa0afb916eda8c427ab))